api/app/src/shared/notifications.ts

```diff
@@ -1,5 +1,7 @@
 import * as Sentry from "@sentry/node";
+import { Extras, ScopeContext } from "@sentry/types";
 import axios from "axios";
+import stringify from "json-stringify-safe";
```

Comment:
I guess we should make this a precedence to use everywhere?

Reply:
Yep, agreed, at least on the projects where we such need - usually when processing an HTTP `request`, but maybe other scenarios as well. At the end of the day, just want to be careful not to add a ton of libs to projects like `react-native` and related.